### PR TITLE
Added maximum pin validation attempts

### PR DESCRIPTION
**Pull request by ashdns**

### CODE REVIEW

File: Source/AppLocker/AppLocker.swift [hunk 1]
```diff
@@ -118,30 +121,31 @@ public class AppLocker: UIViewController {
             isFirstCreationStep = false
             reservedPin = pin
             clearView()
-            submessageLabel.text = "Confirm your pincode"
+            submessageLabel.text = "Confirm pin"
```
Comment (jameszaghini) [inline Source/AppLocker/AppLocker.swift]: This should probably be "PIN" as it's an abbreviation.  Localization as well

**Review by jameszaghini — CHANGES_REQUESTED** [review]
A few strings here need localization

File: Source/AppLocker/AppLocker.swift (hunk 1 above)
Comment (ashdns) [reply Source/AppLocker/AppLocker.swift]: Sure

**Review by jameszaghini — APPROVED** [review]
🧷

**Pull request merged**